### DISCUSSION TITLE
Differences in certs between linkerd cli and linkerd helm method

### DISCUSSION THREAD

**Discussion by rajdhandus** [linkerd/linkerd2#5964] [Q&A]
The cert generated via the linkerd cli and via the step commands given in the docs are different..

Differences:

- Sertial Number (set to 01 in linkerd cli generated cert; set to a long alpha numeric hash like value in step command generated)
- Issuer (set to itself in linkerd cli generated; set to root.linkerd.cluster.local in step command generated)
- Basic Constraint Path Length (set to None in linkerd cli generated; set to 0 in step command generated)
- Subject Key Identifier(not present in linkerd cli generate; present and set to long alpha numeric hash like value in step command generated)

**Comment by olix0r** [comment]
@rajdhandus Thanks for pointing this out! If you have ideas on how to improve the docs, we'd love a PR on the https://github.com/linkerd/website repo
